a3p-integration/proposals/a:upgrade-next/exit-reclaim.test.js

```diff
@@ -0,0 +1,61 @@
+import test from 'ava';
+import { execFile } from 'node:child_process';
+import { promisify } from 'node:util';
+
+const offerId = 'bad-invitation-15'; // cf. prepare.sh
+const from = 'gov1';
+
+test.before(async t => {
+  const $ = async (file, args) => {
+    console.log('$', file, ...args);
+    const { stdout } = await promisify(execFile)(file, args);
+    return stdout.replace(/\n$/, '');
+  };
+  Object.assign(t.context, { $ });
+});
+
+test('exitOffer tool reclaims stuck payment', async t => {
+  const { $ } = t.context;
+
+  const addr = await $('agd', [
```

Comment:
a3p lib has an agd helper, for the future. it even eschews ambient authority
https://github.com/Agoric/agoric-3-proposals/blob/3faecbf11b95da6044ebacd067c5b72f9a45f88f/packages/synthetic-chain/src/lib/core-eval.ts#L57-L60

Reply:
> weird! importing @agoric/synthetic-chain hangs!

This was blocking me for most of the week. Even though I had done it myself more than once, after @turadg suggested running `yarn install` inside the container, the problem cleared up.